lib/node_modules/@stdlib/math/base/special/expm1rel/lib/index.js

```diff
@@ -19,15 +19,15 @@
 'use strict';
 
 /**
-* Relative error exponential.
+* Relative error exponential function.
 *
 * @module @stdlib/math/base/special/expm1rel
 *
 * @example
 * var expm1rel = require( '@stdlib/math/base/special/expm1rel' );
 *
 * var v = expm1rel( 0.0 );
-* // returns TODO
+* // returns 1.0
```

Comment:
We should probably include a couple of other example invocations here. The two in the README with `1.0` and `-1.0` should be fine.

Reply:
You got it.